NOTICE

```diff
@@ -4,58 +4,6343 @@ Copyright 2019-2023 Elasticsearch B.V.
 third-party software developed by the licenses listed below.
 
-=========================================================================
-
-github.com/davecgh/go-spew               0BSD
-github.com/elastic/cloud-sdk-go          Apache-2.0
-github.com/go-openapi/analysis           Apache-2.0
-github.com/go-openapi/errors             Apache-2.0
-github.com/go-openapi/jsonpointer        Apache-2.0
-github.com/go-openapi/jsonreference      Apache-2.0
-github.com/go-openapi/loads              Apache-2.0
-github.com/go-openapi/runtime            Apache-2.0
-github.com/go-openapi/spec               Apache-2.0
-github.com/go-openapi/strfmt             Apache-2.0
-github.com/go-openapi/swag               Apache-2.0
-github.com/go-openapi/validate           Apache-2.0
-github.com/inconshreveable/mousetrap     Apache-2.0
-github.com/oklog/ulid                    Apache-2.0
-github.com/opentracing/opentracing-go    Apache-2.0
-github.com/spf13/afero                   Apache-2.0
-github.com/spf13/cobra                   Apache-2.0
-go.mongodb.org/mongo-driver              Apache-2.0
-gopkg.in/ini.v1                          Apache-2.0
-gopkg.in/yaml.v2                         Apache-2.0
-gopkg.in/yaml.v3                         Apache-2.0
-github.com/magiconair/properties         BSD-2-Clause
-github.com/pkg/errors                    BSD-2-Clause
-github.com/russross/blackfriday/v2       BSD-2-Clause
-github.com/fsnotify/fsnotify             BSD-3-Clause
-github.com/pmezard/go-difflib            BSD-3-Clause
-github.com/sagikazarmark/slog-shim       BSD-3-Clause
-github.com/spf13/pflag                   BSD-3-Clause
-golang.org/x/exp                         BSD-3-Clause
-golang.org/x/sys                         BSD-3-Clause
-golang.org/x/term                        BSD-3-Clause
-golang.org/x/text                        BSD-3-Clause
-github.com/asaskevich/govalidator        MIT
-github.com/blang/semver/v4               MIT
-github.com/cpuguy83/go-md2man/v2         MIT
-github.com/josharian/intern              MIT
-github.com/mailru/easyjson               MIT
-github.com/mitchellh/mapstructure        MIT
-github.com/pelletier/go-toml/v2          MIT
-github.com/sagikazarmark/locafero        MIT
-github.com/sourcegraph/conc              MIT
-github.com/spf13/cast                    MIT
-github.com/spf13/viper                   MIT
-github.com/stretchr/testify              MIT
-github.com/subosito/gotenv               MIT
-go.uber.org/atomic                       MIT
-go.uber.org/multierr                     MIT
-github.com/hashicorp/errwrap             MPL-2.0
-github.com/hashicorp/hcl                 MPL-2.0
-github.com/hashicorp/go-multierror       MPL-2.0-no-copyleft-exception
-github.com/ghodss/yaml                   no license file was found
-
-=========================================================================
+================================================================================
```

Comment:
Do we really need to include the entire license text of each license into NOTICE? Shall we keep the previous file format?

Reply:
No idea. I also liked the previous format better, but that is how all the other elastic repos do it 🤷 

For example: https://github.com/elastic/elastic-agent/blob/main/NOTICE.txt